### ISSUE TITLE
[Bug] `duration` option to `0` don't make it infinite

### ISSUE THREAD

**Issue by Herve-M** [caroso1222/notyf#84]
Hello :-) 

## Describtion
When setting `duration` option to `0`, notification should be visible until the user close/click it (with `dismissible` or not) following the documentation. 
 
### Expected behavior
Setting `duration` option to `0`, notification must be visible until the user close/click it.

### Actual behavior
Setting `duration` option to `0`, notification is visible only for a fraction of ms. 

## How to reproduce & Context
Using https://carlosroso.com/notyf/ under Windows 10 Pro 2004 with: 
* Chromium Version 83.0.4103.61 (Official Build) (64-bit)
* Google Chrome Version 84.0.4147.105 (Official Build) (64-bit)
* Firefox 79.0 (64 bits)

Thanks,
Herve-M

**Comment by shubham2924** [comment]
Yes @Herve-M you're right setting 0ms as duration value doesn't make it infinite.
https://carlosroso.com/notyf/ on their this demo website setting duration to null value makes it infinite
But yes creators eventually need to fix this ASAP.
Thanks,
shubham

**Comment by TECKSPEED** [comment]
Has this been fixed? Or has anything been looked at? I may be looking for an alternative as the bug fixes and such seem lacking

**Comment by KrunchMuffin** [comment]
Same. I was looking for a "sticky" option

**Comment by KrunchMuffin** [comment]
I think something like this could work.

```
private _pushNotification(notification: NotyfNotification) {
  this.notifications.push(notification);
  const duration =
    notification.options.duration !== undefined ? notification.options.duration : this.options.duration;
  if (duration && duration > 0) {
    setTimeout(() => this._removeNotification(notification), duration);
  }
}
```

This will check if the duration is greater than 0 before setting the timeout. If the duration is 0 or less, the timeout will not be set and the notification will not be removed from the DOM.
